c/planarityCommandLine.c

```diff
@@ -425,9 +467,9 @@ int runGraphTransformationTest(char *command, char *infileName, int inputInMemFl
 	// input graph; it will only support "-t(gam)"
 	if (command == NULL || strlen(command) < 3)
 	{
-		// TODO: Update with Issue 18 and 20 to add g and m respectively
+		// TODO: Update with Issue 18 to add g option
 		// ErrorMessage("runGraphTransformationTest only supports -t(gam).\n");
-		ErrorMessage("runGraphTransformationTest only supports -ta.\n");
+		ErrorMessage("runGraphTransformationTest only supports -t(am).\n");

```

Comment:
Error message on line 502 that reads "Failed to perform transformation to produce .g6 output" should have the ".g6" removed because you're not producing g6 output but rather output according to the letter after the -t.